schemas/answers/checkbox.json

```diff
@@ -36,6 +36,9 @@
       "mandatory": {
         "type": "boolean"
       },
+      "no_label": {
```

Comment:
Think we need to make no_label exclusive to label. At the moment it's possible to have a label and a no_label flag, which doesn't make sense

Reply:
Sorted using dependency, thanks!